ledger/streaming/api/src/response.rs

```diff
@@ -13,8 +13,8 @@ pub fn make_subscribe_response(
     let mut proto = SubscribeResponse::new();
     proto.set_result(data.into());
 
-    let result_bytes = proto.get_result().write_to_bytes()?;
-    let signature = signer.try_sign(&result_bytes)?;
+    let digest = get_digest(data);
+    let signature = signer.try_sign(&digest)?;
```

Comment:
We shouldn't really need to sign these unless protobuf requires it? This is a fairly heavy operation that doesn't add much security benefit given downstream components necessarily need to do verification of blocks they receive from peers.